lib/test.js

```diff
@@ -134,7 +138,11 @@ Test.prototype.run = function () {
 	}.bind(this));
 };
 
-Test.prototype.end = function () {
+Test.prototype.end = function (err) {
+	if (arguments.length >= 1 && err) {
```

Comment:
Idea is to only log the assertion if there's some arguments passed. Alternative is have every test log an additional ifError assertion at the end.